docs/section19/Rule 19-1.md

```diff
@@ -0,0 +1,50 @@
+# Section 19 - Rule 19-1  
+**Schema Version:** 0.0.21  
+**Mandatory Rule:** True  
+**Rule ID:** 19-1   
+**Rule Description:** HVAC system coil capacities for the baseline building design shall be oversized by 15% for cooling and 25% for heating..  
+**Rule Assertion:** Options are Pass/Fail/NOT_APPLICABLE     
+**Appendix G Section:** Section G3.1.2.2    
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each HeatingVentilationAirConditioningSystem Data Group  
+
+**Applicability Checks:** 
+
+1. Applies to the baseline with furnaces, heat pumps, and/or DX cooling coils.  
+
+**Function Calls:**  
+1. is_hvac_sys_heating_type_furnace()
+2. is_hvac_sys_heating_type_heat_pump()
+3. is_hvac_sys_cooling_type_DX()  
+
+
+## Rule Logic:  
+**Applicability Check 1:**  Checks each HVAC system for applicability  (does it have a furnace, heatpump, and/or DX cooling) otherwise NOT_APPLICABLE
+- For each hvac system in B_RMR: `for hvac in B_RMR...HeatingVentilationAirConditioningSystem:`    
+    - Reset heating_oversizing_factor: `heating_oversizing_factor = ""`  
+    - Reset cooling_oversizing_factor: `cooling_oversizing_factor = ""`  
+    - Check if the baseline system type has a furnace, heat pump, and/or DX cooling coil, ELSE then rule outcome for this HVAC system is NOT_APPLICABLE: `if is_hvac_sys_heating_type_furnace(B_RMR, hvac.id) == TRUE OR is_hvac_sys_heating_type_heat_pump(B_RMR, hvac.id) ==  TRUE OR is_hvac_sys_cooling_type_DX(B_RMR, hvac.id) == TRUE:`   
+        - Check if the baseline hvac system type has a furnace or heat pump coil at the HVAC system level: `if is_hvac_sys_heating_type_furnace(B_RMR, hvac.id) == TRUE OR is_hvac_sys_heating_type_heat_pump(B_RMR, hvac.id) ==  TRUE:`  
+            - Get the over sizing factor associated with the heating coil: `heating_oversizing_factor = hvac.heating_system.oversizing_factor`     
+        - Else: `Else: heating_oversizing_factor = "n/a"` 
```

Comment:
If possible, we want to avoid change data type of a parameter. In this case, it seems the oversizing factor is assigned to a number (if success) or a string (if failed). Is there a particular reason for that?

Reply:
@weilixu great point. I updated the structure. I was not sure how to reset the number variable, I guess I could use del but I figure this is easy for the RCT to take care of when implementing.